shared-libs/contacts/src/places.js

```diff
@@ -109,8 +117,9 @@ const createPlace = async (place) => {
     return response;
   }
   const placeUUID = response.id;
-
-  contact.place = placeUUID;
+  if (_.isObject(contact)) {
```

Comment:
I'm seeing a strange behavior here: if I try to create a place with a contact, but I fail to create the payload correctly and I post something like: 
```
{
  _id: 'place_with_contact',
  type: 'district_hospital',
  name: 'placeWithoutContact',
  contact: 'angela',
};
```
I get a failed response saying: 
```
{ code: 404, error: 'Failed to find person.' }
```

However if I correct my mistake and try to push the same payload, but with a "good" contact afterwards , like:
```
{
  _id: 'place_with_contact',
  type: 'district_hospital',
  name: 'placeWithoContact',
  contact: {
    name: 'angela'
  }
}
``` 
I get: 
```
{ code: 409, error: 'Document update conflict.' }
```

Here is a gist with the simulation: https://gist.github.com/dianabarsan/1192777672f4d8545ac6495eac90d4d2

Reply:
I think we should cover this use case in some sort of test.

Reply:
Yeah that's gonna happen since the place was getting created with the assumption that a contact is always valid if it's a string. I've updated the code to try and get the contact first if it's a string before creating the place